docs/content/configuration/index.md

```diff
@@ -1312,6 +1312,7 @@ The Druid SQL server is configured through the following properties on the broke
 |`druid.sql.planner.useFallback`|Whether to evaluate operations on the broker when they cannot be expressed as Druid queries. This option is not recommended for production since it can generate unscalable query plans. If false, SQL queries that cannot be translated to Druid queries will fail.|false|
 |`druid.sql.planner.requireTimeCondition`|Whether to require SQL to have filter conditions on __time column so that all generated native queries will have user specified intervals. If true, all queries wihout filter condition on __time column will fail|false|
 |`druid.sql.planner.sqlTimeZone`|Sets the default time zone for the server, which will affect how time functions and timestamp literals behave. Should be a time zone name like "America/Los_Angeles" or offset like "-08:00".|UTC|
+|`druid.sql.planner.serializeComplexValues`|Whether to serialize "complex" output values, false will return the class name instead of the serialized value.|true|
```

Comment:
Commenting for posterity, I think it's fine to set this to true although it changes the old behavior. 

Previously the only complex columns usable in SQL queries were ApproxHistogram and the cardinality/hyperunique aggs, but APPROX_QUANTILE and APPROX_COUNT_DISTINCT would apply a postagg on those and return a numeric value, so a serialized form of the complex column would never be returned previously.